Docs/sphinx/manual/LMeXControls.rst

```diff
@@ -256,6 +256,7 @@ PeleLMeX algorithm
     peleLM.deltaT_tol = 1e-10              # [OPT, DEF=1.e-10] Tolerance of the deltaT solve
     peleLM.evaluate_vars =...              # [OPT, DEF=""] In evaluate mode, list unitTest: diffTerm, divU, instRR, transportCC
     peleLM.do_patch_flow_variables = false # [OPT, DEF=false] Enable user-defined flow variable patching after reading a plot solution file
+    peleLM.do_reset_time = 1               # [OPT, DEF=1] Resets time and nsteps to 0 after restarting from a plot file. (Warning: plot file will be rewritten if not renamed and argument value = 0)
```

Comment:
How about calling this `peleLM.initDataPlt_reset_time` so that it's obvious this setting only applies when initializing from a plt file? And a similar change for `peleLM.do_path_flow_variables`. Also, let's move these control entries to immediately below `peleLM.initDataPlt` in the documentation to make the association even more clear.